torch/onnx/__init__.py

```diff
@@ -116,8 +116,6 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
             input nodes of the graph, in order.
         output_names (list of str, default empty list): names to assign to the
             output nodes of the graph, in order.
-        aten (bool, default False): [DEPRECATED. use operator_export_type] equivalent to
-            setting ``operator_export_type=OperatorExportTypes.ONNX_ATEN``.
         operator_export_type (enum, default OperatorExportTypes.ONNX):
```

Comment:
Thanks for pointing out that the the behavior depends on PYTORCH_ONNX_CAFFE2_BUNDLE. Please update the documentation to something like:
```
operator_export_type (OperatorExportType enum, default None):
    If None, usually defaults to ONNX.
    If PyTorch was built with ``-DPYTORCH_ONNX_CAFFE2_BUNDLE``, defaults to ONNX_ATEN_FALLBACK.
```

Reply:
Thanks, added it.